.azure-pipelines/windows.sh

```diff
@@ -103,14 +97,8 @@ fi
 
 "$HOST_DC" -m$MODEL -g -i run.d
 
-targets=("all")
 args=('ARGS=-O -inline -g') # no -release for faster builds
-if [ "$HOST_DMD_VERSION" = "2.079.0" ] ; then
-    # skip runnable_cxx and unit_tests with older bootstrap compilers
-    targets=("runnable" "compilable" "fail_compilation" "dshell")
-    args=() # use default set of args
```

Comment:
Oh - this definitely accounted for longer runtimes specifically for the bootstrap job - it ran all default permutations, so incl. `-release`, and so doubling the number of compiler test permutations (16 vs. 8).

This is otherwise only enabled for Linux CI jobs: https://github.com/dlang/dmd/blob/9a8302a94133078a869e7fb6285a2b1cf5bf906f/ci/run.sh#L119-L124